client/audio/audio.cpp

```diff
@@ -25,6 +25,7 @@ received a copy of the GNU General Public License along with Freeciv21.
 
 // client
 #include "audio_none.h"
+#include "audio_qt.h"
```

Comment:
Do we need an `#ifdef` for Qt like we have for SDL?

Reply:
Only if we make the plugin optional.

Reply:
Ok cool, that makes sense. This leads to me to another question. At the time we fully pull SDL out would we also take out the "none" plugin and just keep Qt and quit asking for an option in the interface options?

Reply:
I'd keep the option to disable music/sounds, but I guess "none" could be removed because these are separate.